docker-jans-shibboleth/scripts/bootstrap.py

```
@@ -0,0 +1,191 @@
+    def generate_sealer_key(self):
+        logger.info("Generating sealer key")
+        
+        sealer_password = os.environ.get("IDP_SEALER_PASSWORD")
+        if not sealer_password:
+            sealer_password = get_random_chars(24)
+        
+        sealer_file = f"{IDP_HOME}/credentials/sealer.jks"
+        
+        if not os.path.exists(sealer_file):
+            import subprocess
+            subprocess.run([
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)